scripts/main/build.js

```diff
@@ -139,29 +139,29 @@ build.photo = function (data) {
 		}
 
 		if (thumb2x !== '') {
-			thumb2x = `srcset='${thumb2x} 2x'`
+			thumb2x = `class="lazyload" src="dist/images/placeholder.png" data-srcset='${thumb2x} 2x'`
 		}
 
 		thumbnail = `<span class="thumbimg${isVideo ? ' video' : ''}">`;
-		thumbnail += `<img src='${data.thumbUrl}' ` + thumb2x + ` alt='Photo thumbnail' data-overlay='false' draggable='false'>`;
+		thumbnail += `<img class="lazyload" src="dist/images/placeholder.png" data-src='${data.thumbUrl}' ` + thumb2x + ` alt='Photo thumbnail' data-overlay='false' draggable='false'>`;
 		thumbnail += `</span>`
 	} else {
 
 		if (data.small !== '') {
 			if (data.small2x && data.small2x !== '') {
-				thumb2x = `srcset='${data.small} ${parseInt(data.small_dim, 10)}w, ${data.small2x} ${parseInt(data.small2x_dim, 10)}w'`
+				thumb2x = `class="lazyload" src="dist/images/placeholder.png" data-srcset='${data.small} ${parseInt(data.small_dim, 10)}w, ${data.small2x} ${parseInt(data.small2x_dim, 10)}w'`
 			}
 
 			thumbnail = `<span class="thumbimg${isVideo ? ' video' : ''}">`;
-			thumbnail += `<img src='${data.small}' ` + thumb2x + ` alt='Photo thumbnail' data-overlay='false' draggable='false'>`;
+			thumbnail += `<img class="lazyload" src="dist/images/placeholder.png" data-src='${data.small}' ` + thumb2x + ` alt='Photo thumbnail' data-overlay='false' draggable='false'>`;
 			thumbnail += `</span>`
 		} else if (data.medium !== '') {
 			if (data.medium2x && data.medium2x !== '') {
-				thumb2x = `srcset='${data.medium} ${parseInt(data.medium_dim, 10)}w, ${data.medium2x} ${parseInt(data.medium2x_dim, 10)}w'`
+				thumb2x = `class="lazyload" src="dist/images/placeholder.png" data-srcset='${data.medium} ${parseInt(data.medium_dim, 10)}w, ${data.medium2x} ${parseInt(data.medium2x_dim, 10)}w'`
```

Comment:
I don't think you need the `class="lazyload"` here because it is added in line 164. :)